src/Uno.UI/Controls/BindableUIActivityIndicatorView.iOS.cs

```diff
@@ -1,9 +1,16 @@
-﻿using Foundation;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Text;
-using UIKit;
 using Windows.UI.Xaml;
+using CoreGraphics;
+
+#if XAMARIN_IOS_UNIFIED
+using Foundation;
+using UIKit;
+#elif XAMARIN_IOS
+using MonoTouch.Foundation;
+using MonoTouch.UIKit;
+#endif
```

Comment:
This is not needed anymore, you can keep the original usings.

Reply:
Removed across changes